pym/gentoolkit/eclean/search.py

```diff
@@ -563,39 +565,13 @@ def findPackages(
 
     # Dictionary of binary packages to clean. Organized as cpv->[pkgs] in order
     # to support FEATURES=binpkg-multi-instance.
-    dead_binpkgs = {}
+    dead_binpkgs: dict[str, list[str]] = {}
     keep_binpkgs = {}
 
     bin_dbapi = portage.binarytree(pkgdir=pkgdir, settings=var_dbapi.settings).dbapi
     for cpv in bin_dbapi.cpv_all():
         cp = portage.cpv_getkey(cpv)
 
-        # Clean unrecoverable binpkgs
-        if options["clean-invalids"]:
-            try:
-                invalids_path: list[str] = bin_dbapi.bintree.invalids_paths
-                if len(invalids_path) > 0:
-                    dead_binpkgs["invalids"] = invalids_path
-            except AttributeError:
-                invalids: list[str] = bin_dbapi.bintree.invalids
-                print(
-                    pp.error(
-                        "eclean was not able to clean it automatically due to older portage version"
-                    ),
-                    file=sys.stderr,
-                )
-                print(pp.error("portage version >=portage-x.x.x"), file=sys.stderr)
-                print(
-                    pp.error(
-                        "Please remove the following binpkgs manually: %s"
-                        % str(invalids)
-                        .replace("[", " ")
-                        .replace("]", " ")
-                        .replace("'", "")
-                    ),
-                    file=sys.stderr,
-                )
-
```

Comment:
You still have 2 commits. with the first being the commit that did printing in findPackages().

Squash those 2 commits into one, so we see the final change as one commit. WITHOUT the bad coding in between.

Reply:
Done